docs/analytics.md

```diff
@@ -0,0 +1,132 @@
+
+<i>Last updated: Feb 25, 2019</i>
```

Comment:
**PROTIP:** You can replace `<i>...</i>` with `_ ... _`:

```md
_Last updated: Feb 25, 2019_
```

_Last updated: Feb 25, 2019_